sda-commons-server-weld-testing/src/main/java/org/sdase/commons/server/weld/testing/WeldAppRule.java

```diff
@@ -16,7 +16,11 @@
  *   public static final WeldAppRule&lt;AppConfiguration&gt; RULE =
  *       new WeldAppRule&lt;&gt;(Application.class, ResourceHelpers.resourceFilePath("config.yml"));
  * </pre>
+ *
+ * @deprecated migration to JUnit 5 is highly recommended, use {@link WeldAppExtension} as
+ *     alternative
  */
+@Deprecated(forRemoval = true)
```

Comment:
Seems like `forRemoval` was only introduced in Java 11. Breaks for Java 8.

Reply:
fixed